src/config/risingwave.toml

```diff
@@ -1,46 +1,44 @@
-[server]
-heartbeat_interval_ms = 1000
-max_heartbeat_interval_secs = 600
-connection_pool_size = 16
+[batch.developer]
```

Comment:
How about these changes to `risingwave.toml`? Are you tuning for performance? 👀

Reply:
`./risedev c` format for me.